### PR TITLE
Add links to voila, ipywidgets and TLJH

### PR DESCRIPTION
**Pull request by jtpio**

### CODE REVIEW

File: tljh-voila-gallery/tljh_voila_gallery/templates/page.html [hunk 1]
```diff
@@ -50,9 +50,9 @@ <h4 class="text-white">About</h4>
         <div class="col-sm-4 offset-md-1 py-4">
           <h4 class="text-white">Built with:</h4>
           <ul class="list-unstyled">
-            <li><a href="#" class="text-white">Voila</a></li>
-            <li><a href="#" class="text-white">IPywidgets</a></li>
-            <li><a href="#" class="text-white">The littlest JupyterHub</a></li>
+            <li><a href="https://github.com/QuantStack/voila" class="text-white">Voila</a></li>
+            <li><a href="https://github.com/jupyter-widgets/ipywidgets" class="text-white">ipywidgets</a></li>
+            <li><a href="https://github.com/jupyterhub/the-littlest-jupyterhub" class="text-white">The Littlest JupyterHub</a></li>
```
Comment (yuvipanda) [inline tljh-voila-gallery/tljh_voila_gallery/templates/page.html]: This should point to tljh.jupyter.org

Comment (jtpio) [inline tljh-voila-gallery/tljh_voila_gallery/templates/page.html]:
```suggestion
            <li><a href="https://tljh.jupyter.org" class="text-white">The Littlest JupyterHub</a></li>
```

Comment (jtpio) [reply tljh-voila-gallery/tljh_voila_gallery/templates/page.html]: yeah, I guess they could all point to docs instead of GitHub repos

**Comment by pbugnion** [comment]
Ah nice! Don't know how I missed that.

**Pull request merged**